### DISCUSSION TITLE
Color box Footnote

### DISCUSSION THREAD

**Discussion by sondalex** [davidgohel/flextable#488] [Q&A] [answered]
Hi, thank you for developing this great library !

Does the library offer a interface 
to easily add a colour legend footnote ?

Example:

|                |   Model                                                       |
|-------------|--------------------------------------------------------| 
|Constant|       1.18                                                       |
| x1           |$\color{blue}{0.1}$                                   |
| x2           | $\color{red}{1.4}$

![#blue](https://placehold.co/5x5/blue/blue.png) : A first note ...
![#red](https://placehold.co/5x5/red/red.png) : Another note ...

Currently, I hard code LaTeX or HTML depending on my needs but a feature like this
would be really useful.

**Comment by davidgohel** [comment, marked as answer]
Yes, for example with `add_footer_lines`: 

```
library(flextable)
library(grid)
flextable(head(cars)) |> 
  add_footer_lines(
    as_paragraph(
      grid_chunk(
        list(
          grid.circle(gp = gpar(fill="#ec11c2",
                                col = "transparent"))),
        width = .15, height = .15
        ), 
      ": A first note ..."
    )
  )
```

<img width="357" alt="Capture d’écran 2022-11-22 à 10 45 18" src="https://user-images.githubusercontent.com/4331618/203281175-820c6ecb-1530-496b-beb0-c453d23c8ce2.png">

**Reply by sondalex** [reply]
Thank you for this great answer !
